mithril-stm/src/multi_sig.rs

```diff
@@ -643,19 +654,69 @@ mod tests {
             let sk = SigningKey::gen(&mut ChaCha20Rng::from_seed(seed));
             let vk = VerificationKey::from(&sk);
             let sig = sk.sign(&msg);
-            assert!(sig.verify(&msg, &vk).is_ok());
+
+            let result = sig.verify(&msg, &vk);
+            assert!(result.is_ok(), "verify {result:?}");
```

Comment:
This should be enough to see that the result is `Ok`. If not the `unwrap` will help identify the triggered error:
```suggestion
            sig.verify(&msg, &vk).unwrap();
```